src/simulator_core/entities/assets/ates_cluster.py

```diff
@@ -0,0 +1,276 @@
+            PROPERTY_PRESSURE_RETURN: self.solver_asset.get_pressure(1),
+            PROPERTY_TEMPERATURE_SUPPLY: self.solver_asset.get_temperature(0),
+            PROPERTY_TEMPERATURE_RETURN: self.solver_asset.get_temperature(1),
+        }
+        self.output.append(output_dict)
+
+    def _init_rosim(self) -> None:
+        """Function to initailized Rosim from XML file."""
+        xmlfile = os.path.join(path, 'bin/sequentialTemplate_v0.4.2_template.xml')
+        with open(xmlfile, 'r') as fd:
+            xml_str = fd.read()
+
+        # overwrite template value with ESDL properties
+
+        MODEL_TOP = self.aquifer_depth - 100
+        AQUIFER_THICKNESS = self.aquifer_thickness
+        NZ_AQUIFER = math.floor(AQUIFER_THICKNESS / 2)
+        NZ = NZ_AQUIFER + 8
+        AQUIFER_TOP = self.aquifer_depth
+        AQUIFER_BASE = self.aquifer_depth + self.aquifer_thickness
+        SURFACE_TEMPERATURE = self.aquifer_mid_temperature - 0.034 * (
+            self.aquifer_depth + self.aquifer_thickness / 2)
+        AQUIFER_NTG = self.aquifer_net_to_gross
+        AQUIFER_PORO = self.aquifer_porosity
+        AQUIFER_PERM_XY = self.aquifer_permeability
+        AQUIFER_PERM_Z = AQUIFER_PERM_XY / self.aquifer_anisotropy
+        SALINITY = self.salinity
+        WELL2_X = self.well_distance + 300
+        CASING_SIZE = self.well_casing_size
+
+        xml_str = xml_str.replace('$NZ$', str(NZ))
+        xml_str = xml_str.replace('$MODEL_TOP$', str(MODEL_TOP))
+        xml_str = xml_str.replace('$TIME_STEP_UNIT$', str(2))
+        xml_str = xml_str.replace('$WELL2_X$', str(WELL2_X))
+        xml_str = xml_str.replace('$AQUIFER_TOP$', str(AQUIFER_TOP))
+        xml_str = xml_str.replace('$AQUIFER_BASE$', str(AQUIFER_BASE))
+        xml_str = xml_str.replace('$CASING_SIZE$', str(CASING_SIZE))
+        xml_str = xml_str.replace('$SURFACE_TEMPERATURE$', str(SURFACE_TEMPERATURE))
+        xml_str = xml_str.replace('$SALINITY$', str(SALINITY))
+        xml_str = xml_str.replace('$NZ_AQUIFER$', str(NZ_AQUIFER))
+        xml_str = xml_str.replace('$AQUIFER_THICKNESS$', str(AQUIFER_THICKNESS))
+        xml_str = xml_str.replace('$AQUIFER_PORO$', str(AQUIFER_PORO))
+        xml_str = xml_str.replace('$AQUIFER_NTG$', str(AQUIFER_NTG))
+        xml_str = xml_str.replace('$AQUIFER_PERM_XY$', str(AQUIFER_PERM_XY))
+        xml_str = xml_str.replace('$AQUIFER_PERM_Z$', str(AQUIFER_PERM_Z))
+
+        temp_xmlfile_path = os.path.join(path, 'bin/ates_sequential_temp.xml')
+        with open(temp_xmlfile_path, 'w') as temp_xmlfile:
+            temp_xmlfile.write(xml_str)
+
+        xmlfilejava = javaioFile(temp_xmlfile_path)
+        self.rosim = RosimSequential(xmlfilejava, False, 2)
+
+    def _run_rosim(self) -> None:
+        """Function to calculate storage temperature after injection and production."""
+        volume_flow = self.mass_flowrate * 3600 / 1000
+        timestep = 1  # HARDCODED to 1 hour
```

Comment:
The time step should come from above class, this is something the user will give in, Ok to do this now like this, but please make an issue that we need to change this

Reply:
i have created issue #151